modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -656,21 +656,22 @@ definition {
 				whenAction = "On After Add");
 		}
 
-		task ("When: An entry is added") {
+		task ("When an entry is added") {
 			ObjectAdmin.goToCustomObject(objectName = "CustomObject175197");
 
 			ObjectPortlet.addSingleFieldEntryViaUI(entry = "Entry Test");
 		}
 
-		task ("Then: The action is triggered and the notification is sent") {
+		task ("Then the action is triggered and the notification is sent") {
 			ApplicationsMenu.gotoPortlet(
 				category = "Notifications",
 				panel = "Control Panel",
 				portlet = "Queue");
 
 			AssertElementPresent(
-				key_objectName = "CustomObject175197",
-				locator1 = "ObjectNotifications#OBJECT_QUEUE_NAME");
+				key_entryLabel = "CustomObject175197",
+				key_entryStatus = "Sent",
+				locator1 = "ObjectNotifications#VIEW_NOTIFICATION_ENTRIES");
```

Comment:
Could use the MockMock to assert the entry is sent?

For all testcases that use MockMock is necessary to add the property: `property test.smtp.server.enabled = "true";`

There are some examples in Poshi.